marine-examples/sqlite/README.md

```diff
@@ -1,115 +1,112 @@
 # SQLite Example
 
+
+
 ## Overview
 
-[SQLite](https://www.sqlite.org/index.html) is an ubiquitous persistence solution also available on the Fluence stack. Unlike basic compute modules, using SQLite in a service is a little more intricate. In order to create the service, we need two dependencies: An SQLite Wasm module and an SQLite adapter, where the adapter is a [WASM IT compliant](https://crates.io/crates/marine-sqlite-connector) [sqlite](https://github.com/stainless-steel/sqlite) package implementation. The SQLite Wasm module can be obtain from the repo as a [release](https://github.com/fluencelabs/sqlite/releases/tag/v0.15.0_w) or compiled from [code](https://github.com/fluencelabs/sqlite).  For the purpose of this example, we use the release version -- see `build.sh`.
 
-As a result, our service is comprised of two modules: the SQLite Wasm module and our sqlite module. 
 
-## Build
+[SQLite](https://www.sqlite.org/index.html) is ubiquitous persistence solution also available in the Fluence stack. Unlike basic compute modules, using SQLite in a service is a little more intricate. In order to create a service, one needs two dependencies:
+- SQLite Wasm module that one can get either in a form of a [released module](https://github.com/fluencelabs/sqlite/releases/tag/v0.17.1_w) or build it from scratch following [this](https://github.com/fluencelabs/sqlite) readme.
+- SQLite connector that is [WASM IT compliant](https://crates.io/crates/marine-sqlite-connector) [sqlite](https://github.com/stainless-steel/sqlite) package implementation.
+
+For the purpose of this example, we use the release version -- see `build.sh`.
+
+
+
+As a result, our service is comprised of two modules: the SQLite Wasm module and our sqlite module.
+
+
+
+## How to build
+
+
+
+To build the Wasm modules needed run the following command:
+
 
-To build the required artifacts, please run the following command:
 
 ```
-./build.sh
+
+$build.sh
+
 ```
 
-Upon a successful build it results a couple of wasm modules:
+
+
+Upon a successful build there will be a pair of wasm modules:
+
+
 
 ```
-➜  sqlite git:(main) ls artifacts
-sqlite3.wasm     sqlite_test.wasm
+$ls artifacts
+
+sqlite3.wasm sqlite_test.wasm
 ```
 
-## Run
 
-To check how it works, we need to run `mrepl`:
 
+## How to use
+
+
+
+One needs to install marine and mrepl Rust crates to try this example. Here is a simple 'CREATE-INSERT-SELECT' scenario output using the artifacts of this repo.
+
+```bash
+$cargo install marine mrepl
```

Comment:
this assumes you are in nightly. better make it explicit. also, we are currently constraint to nightly nightly-2022-12-06-x86_64-apple-darwin or older.

Reply:
This is a very good point indeed.
@mikevoronov should we make this explicit with the current state of affairs, versions, modules?